postgres/migrations/00002_create_archer.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
pwede mo na pag samahahin sa single file with fighters table dahil di pa nmn na dedeploy sa prod. pero pag deployed na goods to